app/general/models.py

```diff
@@ -167,5 +167,7 @@ class Meta:
             GinIndex(fields=["search_vector"]),
         ]
 
+        permissions = [("can_edit_fulltext", "Can edit document fulltext (used for search)")]
```

Comment:
I assume the second part must be marked for translation with _("Can ..."). It might require regeneration of the migration.

Reply:
Hmm, I've never declared permissions before - is it even possible for them to be translated?

Reply:
I assumed this would work to follow the pattern of other similar tuples, but alas, no. It is not currently possible with some open discussions and bugs. Sorry for the noise.